5_databases/week2/Part3.sql

```diff
@@ -0,0 +1,20 @@
+--Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT *
+FROM task JOIN `user` on email LIKE "%@spotify.com"
+
+--Get all the tasks for 'Donald Duck' with status 'Not started'
+
+SELECT task.title, user.name, status_id
```

Comment:
You want to include user_task table to connect the many_to_many relations here to get the correct response